tests/main/mount-ns/task.yaml

```diff
@@ -144,6 +148,14 @@ prepare: |
     su root -c "snap run test-snapd-mountinfo-core18" >PER-SNAP-18.raw.txt
     su test -c "snap run test-snapd-mountinfo-core18" >PER-USER-18.raw.txt
 
+    echo "Collect mountinfo from the host after the test"
```

Comment:
it's a little bit weird that this says "after the test" when it is being run in the prepare section... not that there's much to be done about this, I understand why this is done this way, I guess more just speaking to the wind as it were about this oddity

Reply:
I see what you mean. I can change it to say "after executing snap apps".

Reply:
Updated in https://github.com/snapcore/snapd/pull/7668